### PR TITLE
First pass of slack confirm page

### PR DESCRIPTION
**Pull request by andrewconner**
@cvializ Suuuuuuper rough, not pretty yet (and markup can improve). Want to open this for you to check out and to get it QA-able sooner.

### CODE REVIEW

**Comment by andrewconner** [comment]
The way it looks currently:

File: kifi-backend/modules/shoebox/angular/src/route.js [hunk 1]
```diff
@@ -213,6 +213,21 @@ angular.module('kifi')
...
         activenav: 'integrations'
       })
+      .state('orgProfile.settings.integrationsSlackConfirm', {
+        url: '/integrations/slack-confirm',
+        controller: 'SlackConfirmCtrl',
+        templateUrl: 'teamSettings/slackConfirm.tpl.html',
+        activetab: 'settings',
+        activenav: 'integrations',
+        resolve: {
+          profile: [
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/route.js]: I think `profile` already be injected implicitly from the parent view (`orgProfile.settings`). Just ask for `'profile'` in the list of dependency injections in `SlackConfirmCtrl`.

If you already tried that and it didn't work then carry on.

File: kifi-backend/modules/shoebox/angular/src/teamSettings/slackConfirm.tpl.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  -->
+
+  <h2>😻 Congrats, we’re syncing your channels now 😻</h2>
+
+  <p>We’re pulling in content to ​the <a ui-sref="orgProfile.libraries"><span ng-bind="profile.organization.name"></span> team page</a> — it make take a bit. Full search may take up to an hour because we’re downloading all your links so you can search them.</p>
+
+  <div ng-if="hasInstalled">
+    <p>Since you already have our browser extensions (high five!), encourage your team to get it as well so they can benefit from the links kept here. Send them to this link to sign up with Slack and automatically join this Kifi team:</p>
+    <div class="kf-slack-confirm-team-link-wrapper">
+      <input type="text" ng-model="teamLink" class="kf-textbox kf-slack-confirm-team-link" clipboard readonly text="teamLink" ng-click="showCopied()">
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/slackConfirm.tpl.html:26]: `ng-model="teamLink"` and `text="teamLink"`? What does the `text` attr do? Usually I see `value` used, but I thought `ng-model` would set that

**Comment by cvializ** [comment]
lgtm :raised_hands:

File: kifi-backend/modules/shoebox/angular/src/route.js [hunk 3]
```diff
@@ -226,6 +226,13 @@ angular.module('kifi')
         activetab: 'settings',
         activenav: 'integrations'
       })
+      .state('orgProfile.settings.integrationsSlackConfirm', {
+        url: '/integrations/slack-confirm?&:slackTeamId',
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/route.js]: what does the `?&:` do? I would have used just `?slackTeamId`

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/route.js]: You would have been better than me

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/route.js]: lol I don't know about that, those emoji kitties are ~~on point~~
edit: on fleek

**Pull request merged**